examples/effortless_config/policyfile_cookbook_pattern/.kitchen.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This should be kitchen.yml not .kitchen.yml

Reply:
Good catch I'll change that.

Reply:
:+1: We've been moving away from hidden config files.